src/components/frame/Extensions/pages/MultiSigPage/MultiSigPageSetup.tsx

```diff
@@ -115,17 +197,32 @@ const MultiSigPageSetup: FC<MultiSigPageSetupProps> = ({ extensionData }) => {
     }
   }, [extensionData, pathname, navigate]);
 
-  const getSetThresholdPayload = () => {
-    if (thresholdType === MultiSigThresholdType.MAJORITY_APPROVAL) {
-      return {
-        colonyAddress,
-        threshold: 0,
-      };
-    }
+  const getSetThresholdsPayload = () => {
```

Comment:
just thinking if it still makes sense for this to be an object, since we are using an array on the block-ingestor and also `gql` returns an array too.